install_files/securedrop-app-code/DEBIAN/postinst

```diff
@@ -24,8 +24,17 @@ case "$1" in
     chown www-data:www-data /var/www/document.wsgi
     chown www-data:www-data /var/www/source.wsgi
 
-    aa-complain /etc/apparmor.d/usr.sbin.tor
-    aa-complain /etc/apparmor.d/usr.sbin.apache2
+    a2dissite 000-default
```

Comment:
Mind adding a comment here? Also, can't remember - *why* did we agree it was better to do this in the postinst, instead of in the Ansible playbook?